ajax/libs/jquery.touch/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "jquery.touch",
+  "filename": "jquery.touch.min.js",
+  "description": "Adds touch gesture events to jQuery",
+  "keywords": [
+    "jquery",
+    "touch"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ajlkn/jquery.touch"
+  },
+  "license": "MIT",
+  "author": "aj <@ajlkn> (https://aj.lkn.io/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ajlkn/jquery.touch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.touch*"
```

Comment:
@dakshshah96 please use `jquery.touch*.+(js|css|map)`.
This is a better pattern matcher allowing selection of multiple files ( ref: [node-glob](https://github.com/isaacs/node-glob#glob-primer))
ref: https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md
Thank you.